cypress/integration/pages/testsForAllPages.js

```diff
@@ -4,8 +4,12 @@ import appConfig from '../../../src/app/lib/config/services';
 import describeForEuOnly from '../../support/helpers/describeForEuOnly';
 import useAppToggles from '../../support/helpers/useAppToggles';
 
-const testsForAllPages = ({ service, pageType }) => {
-  describe('Always tests', () => {
+export const testsToAlwaysRunForAllPages = ({ service, pageType }) => {
```

Comment:
Where maybe a little of https://github.com/bbc/simorgh/pull/3364 would go @sareh I'm thinking canonical test